src/page/login/login.tsx

```diff
@@ -0,0 +1,93 @@
+import React, {Props} from 'react';
+import {View, Text, TextInput, TouchableHighlight} from 'react-native';
+import {styles} from './loginCss';
+import Reactotron from 'reactotron-react-native';
+// import {FontAwesomeIcon} from '@fortawesome/react-native-fontawesome';
+// import {faCoffee} from '@fortawesome/free-solid-svg-icons';
+// import Logo from '../../resource/images/logo.svg';
+
+const Login: React.FC = (Props: Props) => {
+  /**
+   * login
+   */
+  function login() {
```

Comment:
In the future please move all async actions in redux middleware.
Install and setup `redux-saga`, please.
Let's use next template for actions to fetch data:
NAME_OF_THE_ACTION + SUCCESS / REQUEST / FAILURE

i.e.
```
cosnt LOGIN_REQUEST = 'LOGIN_REQUEST'
const LOGIN_SUCCESS ='LOGIN_SUCCESS'
cosnt LOGIN_FAILURE = 'LOGIN_FAILURE'

```

1) _REQUEST action will set `loading: true` in redux state and invoke `saga` with fecth request inside
2) saga will dispatch _SUCCESS action on success and _FAILURE on request fail
3) _SUCCESS or _FAILURE will change `loading:false` in redux state